.github/scripts/codespell-ignore.txt

```diff
@@ -1,16 +1,38 @@
 # Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
 #
 # SPDX-License-Identifier: curl
-clen
-te
-wont
-statics
-nome
-wast
-numer
+admi
+anc
 anull
-inout
-msdos
 ba
-fo
+bu
+clen
+CNA
+couter
+dout
 ede
+fo
+htpt
+indx
+inout
+nome
+numer
+parm
+Nam
+PASE
+perfec
+pres
+rin
+seh
+ser
+statics
+strat
+te
+ths
+ttests
+unexpect
```

Comment:
There is *one* use of `unexect` in the entire git repository and that is a variable in a perl script. It's not even a typo, its a variable name. I don't think we should whitelist this word for the entire repo because of this.

Reply:
Agreed it's better to rename that variable.